### PR TITLE
add a Rakefile for ZAP pipeline builds/deployments

### PR DESCRIPTION
**Pull request by afeld**

### CODE REVIEW

**Comment by afeld** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;D3cQA4FW&#x2F;126-write-a-script-to-deploy-zap-pipeline-to-production"> Write a script to deploy ZAP pipeline to production</a>

File: .rubocop.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # based on https://github.com/18F/C2/blob/2032fcb4de80ad23d9b81885be20985e99052b35/.rubocop.yml
 Metrics/LineLength:
   Enabled: false
+Style/AlignParameters:
+  EnforcedStyle: with_fixed_indentation
```
Comment (ctro) [inline .rubocop.yml:5]: I'm down with this.

File: README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Uploading a pipeline is done via the `fly set-pipeline` command, which is includ
 
 ### ci.cloud.gov
 
-Running pipelines on ci.cloud.gov is identical to running pipelines locally, with the exception that you will need to log in to ci.cloud.gov. Assuming you have permissions:
+Running pipelines on ci.cloud.gov is identical to running pipelines locally, with the exception that you will need to log in to ci.cloud.gov. Assuming you have permissions, here is how you would deploy the [ZAP pipeline](pipelines/zap/):
```
Comment (ctro) [inline README.md]: Is it worth mentioning that this deploys from github's HEAD of the branch that you are on locally?   Could be confusing.

File: pipelines/zap/Rakefile [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    '--pipeline', 'zap'
+end
+
+desc "Open the pipeline in a browser."
+task open: :verify_target do
+  sh 'open', @origin
+end
+
+desc "Build and update the pipeline on the given target."
+task deploy: [:build, :unpause, :open] do
```
Comment (ctro) [inline pipelines/zap/Rakefile]: does this need to `:set` too?

**Comment by afeld** [comment]
Updated! Also switched it to use ci-tooling.cloud.gov, per https://trello.com/c/jWUypPQv/129-migrate-to-new-concourse-instance.

**Comment by ctro** [comment]
:+1:  Looks good to me! @DavidEBest you wanna peek at this one?

**Comment by DavidEBest** [comment]
This is really slick. 👍

**Pull request merged**